ocs_ci/ocs/ocp.py

```diff
@@ -506,3 +509,41 @@ def switch_to_default_rook_cluster_project():
         bool: True on success, False otherwise
     """
     return switch_to_project(defaults.ROOK_CLUSTER_NAMESPACE)
+
+
+def rsync(src, dst, node, dst_node=True, extra_params=""):
+    """
+    This function will rsync source folder to destination path.
+    You can rsync local folder to the node or vice versa depends on
+    dst_node parameter. By default the rsync is from local to the node.
+
+    Args:
+        src (str): Source path of file/folder to rsync.
+        dst (str): Destination path where to rsync.
```

Comment:
If I understand it correctly, it expects absolute path here? I would mention it in the doc.